src/main.rs

```diff
@@ -360,27 +345,24 @@ fn main() {
         println!();
         return;
     }
-    let caps = matches.value_of("caps").unwrap_or("true");
-    let abold = matches.value_of("bold").unwrap_or("true");
+    let current_user = if !matches.is_present("no-user") || !matches.is_present("no-shell") {
+        Passwd::current_user()
+    } else {
+        None
+    };
+    let abold = !matches.is_present("no-bold");
```

Comment:
`abold`?

Reply:
That has to do with kiedtl's function. They already had it set up like this:

```Rust
if abold != false {
        title_str = make_bold(&title_str);
}
```

So I decided to just not touch it. The variable is just an extra character long than it needs to be, so I figured no harm done. I can change it though.

Reply:
It should probably say `bold` unless I'm missing something. It's easier to just fix stuff when you notice it. Also, `if bold` instead of `if bold != false`, of course.